server/data/restClient.ts

```diff
@@ -156,7 +156,7 @@ export default class RestClient {
             reject(error)
           } else if (response) {
             const s = new Readable()
-            // eslint-disable-next-line no-underscore-dangle,@typescript-eslint/no-empty-function,no-empty-function
+            // eslint-disable-next-line no-underscore-dangle,no-empty-function
```

Comment:
Although you'll still need a disable line for the underscore, it might be nicer to add a comment inside the empty function and lose the 'no-empty-function' directive.

Reply:
A comment explaining _why_ the function is empty would be good, but I'll be honest I'm not entirely sure I get this piece of code, so I can't add any insight! I'll leave this one as is, until it gets revisited with some more enlightenment added 😄